.github/workflows/overview-dag.yml

```
@@ -0,0 +1,159 @@
+      - name: Prepare comment
+        id: prep
+        run: |
+          RAW="${{ needs.publish-preview.outputs.raw_url }}"
+          RUN_URL="${{ needs.build-preview.outputs.run_url }}"
+          SVG_B64="${{ needs.build-preview.outputs.svg_b64 }}"
+          if [ -n "$RAW" ]; then
+            cat <<EOF > $RUNNER_TEMP/cmt.md
+### Your GitHub Issues as a DAG
+
+Preview (same-repo PR):
+<img src="$RAW" alt="Overview DAG"/>
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)